test/modules/linear_boltzmann_solvers/transport_adjoint/response_2d_3.lua

```diff
@@ -162,15 +162,15 @@ function ResponseFunction(xyz, mat_id)
 end
 response_func = opensn.LuaSpatialMaterialFunction.Create({ lua_function_name = "ResponseFunction" })
```

Comment:
is there some sort of default ot be a ResponseFunction? If so, should be just be a bit more generic?

Reply:
I'm not quite sure what you mean.

Reply:
IT's about this part: `lua_function_name = "ResponseFunction"`

Reply:
This comment was again typed up yesterday but left in a "pending" state in a review.

The argument `lua_function_name` must correspond to the name of a defined Lua function in the script. In this case, `ResponseFunction` is the name of the function defined directly above this line. If it was called something else, the argument would have to be changed. By default `lua_function_name` is empty. In the previous version, this resulted in the default behavior of a unit source within the logical volume (left over from the original adjoint implementation to default to a volumetric QoI). In the current implementation, if `lua_function_name` is empty, a `group_strength` vector must be supplied. When a `group_strength` vector is specified, this acts as a constant source within the logical volume and/or block IDs.

Reply:
Oh, I see. The function is only used to define the source for the adjoint problem. The function is defined after the forward solve is done, so it is not used there.